specification/compute/resource-manager/readme.md

```diff
@@ -207,6 +207,11 @@ directive:
       - TrackedResourcePatchOperation
     reason:
       - ACS service is deprecated so a PATCH endpoint won't be implemented
+  - where:
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/cloudServices/{cloudServiceName}/roleInstances/{roleInstanceName}/remoteDesktopFile"].get
+    suppress:
```

Comment:
Why is this needed?

Reply:
the reason is listed below: reason: The API response has binary format and file type which is valid Swagger format. However, the example must be a JSON file which does not support specifying this response format.

The team owning the tests told me to supress it as they don't have a solution